controls/views.py

```diff
@@ -515,6 +517,14 @@ def as_json(self):
             },
         }
 
+        # Add component's tags if they exist
+        if len(self.element.tags.all()) > 0:
```

Comment:
change to:
1. if self.element.tags.exists()
2. if self.element.tags.count() > 0

Reply:
Implemented `self.element.tags.exists()`